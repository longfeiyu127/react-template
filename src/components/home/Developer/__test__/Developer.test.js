import React from 'react';
import renderer from 'react-test-renderer';
import Developer from '../Developer';

describe('<Developer />', () => {
  it('component render', () => {
    let developer = renderer.create(<Developer />).toJSON();
    expect(developer).toMatchSnapshot();

    developer = renderer.create(<Developer />).toJSON();
    expect(developer).toMatchSnapshot();
  });
});
