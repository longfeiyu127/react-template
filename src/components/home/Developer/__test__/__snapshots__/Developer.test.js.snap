// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Developer /> component render 1`] = `
<h2>
  开发者
</h2>
`;

exports[`<Developer /> component render 2`] = `
<h2>
  开发者
</h2>
`;
